.github/scripts/run_consensus.sh

```diff
@@ -17,6 +17,15 @@ MIN_VALIDATOR_COUNT=4
 
 export NODE_IMAGE
 
+if [[ -z $(docker inspect ${NODE_IMAGE} | grep "No such object") ]]; then
```

Comment:
@fixxxedpoint , I had to change this condition as `grep aleph-node:latest` didn't match any - the output of `docker images` is:
```sh
~/aleph/aleph-node$ docker images
REPOSITORY                                  TAG       IMAGE ID       CREATED          SIZE
aleph-node                                  latest    006771953aa3   10 minutes ago   291MB
```
hence `grep aleph-node:latest` didn't catch it. Now we call `docker inspect` which respects name and tag.